monitoring/snippets/v3/cloud-client/snippets.py

```diff
@@ -58,7 +73,12 @@ def delete_metric_descriptor(descriptor_name):
     # [END monitoring_delete_metric]
 
 
-def write_time_series(project_id):
+def write_time_series(project_id: str) -> None:
```

Comment:
Can you please redo the sample to return a value and use it in the test instead of output parsing?

Reply:
The use of `write_time_series` is different from `custom_metric_descriptor`. It is used in tests but not its return values.